tidb-server/main.go

```diff
@@ -389,6 +389,8 @@ func setGlobalVars() {
 	if cfg.TiKVClient.GrpcConnectionCount > 0 {
 		tikv.MaxConnectionCount = cfg.TiKVClient.GrpcConnectionCount
 	}
+	tikv.GrpcKeepAliveTime = time.Duration(cfg.TiKVClient.GrpcKeepAliveTime) * time.Second
```

Comment:
I think we should check where the configuration is valid. For example, the configured time duration should be greater than zero.

Reply:
It seems that other configurations are not checked as well, except for the 1 config above (GrpcConnectionCount). I think it would be better to leave to another PR to do this.

Reply:
OK, could you file an github issue about this?

Reply:
@zz-jason Yes! I just created one: https://github.com/pingcap/tidb/issues/7103